src/feautures/MovieList/styled.js

```diff
@@ -0,0 +1,17 @@
+import styled from "styled-components";
+
+export const StyledMovies = styled.div`
+    display: grid;
+    grid-template-columns: auto auto auto auto;
```

Comment:
try using:

- `grid-template-columns: repeat(auto-fill, 324px);`
- `justify-content: center;`

For it to work make sure you set Section's side paddings to 0 💪